### ISSUE TITLE
This project requires a license

### ISSUE THREAD

**Issue by Jivanvl** [ecomfe/echarts-doc#43]
Probably more of a question, but shouldn't this project have the same license as https://github.com/apache/incubator-echarts?

**Comment by Ovilia** [comment]
Yes. We probably should provide a license file for this project.
